zerver/models/groups.py

```diff
@@ -17,7 +17,7 @@ class SystemGroups:
     NOBODY = "role:nobody"
 
 
-class UserGroup(models.Model):  # type: ignore[django-manager-missing] # django-stubs cannot resolve the custom CTEManager yet https://github.com/typeddjango/django-stubs/issues/1023
```

Comment:
The linter gave error that the `# type: ignore` is not required. I am not sure why it does so since that was added before we added the `can_mention_group` setting.